t/05-scheduler-full.t

```diff
@@ -90,30 +90,30 @@ sub create_worker {
 }
 
 subtest 'Scheduler worker job allocation' => sub {
-    # Step 1
-    my $allocated = scheduler_step();    # Will try to allocate to previous worker and fail!
+    # try to allocate to previous worker (supposed to fail)
```

Comment:
I suggest to actually change this comment into `note(...)`

Reply:
Done, I'll push that later with the other amendments.